pulsar-sql/presto-distribution/src/assembly/assembly.xml

```diff
@@ -40,6 +40,11 @@
             <outputDirectory>bin/</outputDirectory>
             <fileMode>644</fileMode>
         </file>
+        <file>
+            <source>${basedir}/../java-version-trim-agent/target/java-version-trim-agent.jar</source>
```

Comment:
the problem with this is that it doesn't have a project dependency and the build fails in CI. To fix the issue, you should add this dependency to `presto-distribution/pom.xml`:
```
    <dependency>
      <groupId>${project.groupId}</groupId>
      <artifactId>java-version-trim-agent</artifactId>
      <version>${project.version}</version>
      <scope>provided</scope>
    </dependency>
```

Reply:
Thanks, fixed.